File: Algorithms/Search/2_LinearSearch/Python/linearSearch.py
def linearSearch(array, element):
    for i in range(len(array)):
        if array[i] == element:
            return i


    return -1


array = [0, 1, 2, 3, 4, 5, 6, 7]
element = 5

print("Array:", array)

index = linearSearch(array, element)
if index != -1:
    print(f"Elemento encontrado no índice {index}")
else:
    print("Elemento não encontrado")

File: Algorithms/Search/2_LinearSearch/Rust/linear_search.rs
fn linear_search(arr: Vec<i32>, target: i32) -> i32 {
    for i in 0..arr.len() {
        if arr[i] == target {
            return i as i32;
        }
    }
    -1
}

fn main() {
    let arr = vec![1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
    let target = 5;
    let result = linear_search(arr, target);
    println!("Índice do Elemento {}: {}", target, result);
}
